ocpp/v16/call.py

```diff
@@ -122,10 +125,46 @@ class SendLocalListPayload:
     local_authorization_list: List = field(default_factory=list)
 
 
+@dataclass
+class ChargingSchedulePeriod:
+    """Charging schedule period structure defines a time period
+     in a charging schedule, as used in: ChargingSchedule."""
+    start_period: int
+    limit: Decimal
```

Comment:
`Decimal` aren't JSON-serializable. Can you use regular floats here?

Reply:
Yup no problem I'll change that. I had thought, perhaps mistakenly, that as `ocpp.messages._DecimalEncoder` is used in the serialisation of `Call` it would be serialised correctly.

Reply:
You have a fair point. I forgot about that. However, I still suggest to change to floats. I don't see any big advantages to use `Decimal` of `float`. In that case I prefer `float` as this type is way more used. Or do you have any strong argument to use `Decimal`?

Reply:
@grutts did you have time to check this?

Reply:
@tropxy ah I forgot to address this. I've changed the types to float in the latest commit.

I also moved the dataclasses to a new `charging_profile` module and added `ChargingProfile` as an acceptable type to the `RemoteStartTransaction` payload (alongside `Dict` and `None`)